### DISCUSSION TITLE
Dashboard View

### DISCUSSION THREAD

**Discussion by OSXGroupFL** [cockpit-project/cockpit#16555] [General]
I just upgraded to 252 from 215 and the Dashboard view is gone. The was one of the big reasons I wanted to use Cockpit was a single pane of glass where I could see all the servers. Is it getting a redesign or is it just gone?

**Comment by martinpitt** [comment]
c-dashboard got removed in [234](https://cockpit-project.org/blog/cockpit-234.html) and replaced with the builtin host switcher at the top left, and [grafana integration](https://cockpit-project.org/blog/pcp-grafana.html) for proper graphs.
